client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -187,16 +206,25 @@ class TrsConnection {
   std::unique_ptr<grpc::ClientReaderInterface<com::vmware::concord::thin_replica::Data>> state_stream_;
   std::unique_ptr<grpc::ClientContext> hash_context_;
   std::unique_ptr<grpc::ClientReaderInterface<com::vmware::concord::thin_replica::Hash>> hash_stream_;
+  std::map<RequestId,
+           std::pair<std::unique_ptr<grpc::ClientContext>,
+                     std::unique_ptr<
+                         grpc::ClientReaderInterface<vmware::concord::replicastatesnapshot::StreamSnapshotResponse>>>>
+      rss_streams_;
+  std::mutex rss_streams_mutex_;
+  std::atomic_uint64_t current_req_id_{1};
 
   // gRPC connection
   std::shared_ptr<grpc::Channel> channel_;
   std::unique_ptr<com::vmware::concord::thin_replica::ThinReplica::StubInterface> stub_;
```

Comment:
I suggest to rename this to `trc_stub_`.

Reply:
thanks @robem, I will change this in next commit.